int8 id
LightControlType command